features/attached_commands.feature

```diff
@@ -370,14 +370,30 @@ Feature: Command behaviour when attached to an UA subscription
         And I attach `contract_token` with sudo
         And I run `ua status --all` with sudo
         Then stdout matches regexp:
-            """
-            SERVICE       +ENTITLED  STATUS    DESCRIPTION
-            cc-eal        +no
-            """
-        When I run `ua --version` as non-root
-        Then I will see the uaclient version on stdout with features ` +disable_auto_attach +machine_token_overlay -other`
-        When I run `ua version` as non-root
-        Then I will see the uaclient version on stdout with features ` +disable_auto_attach +machine_token_overlay -other`
+        """
+        SERVICE       +ENTITLED  STATUS    DESCRIPTION
+        cc-eal        +no
+        """
+        And stdout matches regexp:
+        """
+        FEATURES
+        \+disable_auto_attach
+        \+machine_token_overlay
+        -other
+        """
+        When I run `ua status --all` as non-root
+        Then stdout matches regexp:
+        """
+        SERVICE       +ENTITLED  STATUS    DESCRIPTION
+        cc-eal        +no
+        """
+        And stdout matches regexp:
+        """
+        FEATURES
+        \+disable_auto_attach
+        \+machine_token_overlay
+        -other
```

Comment:
Since this no longer has to fit on one line in the version, I think we can be more verbose.
```suggestion
        disable_auto_attach: True
        machine_token_overlay: /path/to/overlay
        other: other
```

Reply:
ack agree
done